### ISSUE TITLE
Add account and network API methods

### ISSUE THREAD

**Issue by friedger** [blockstack/blockstack-android#111]
The following blockstack.js methods should be added:
```
getNamePrice
getNamespacePrice
getGracePeriod
getNamesOwned
getNamespaceBurnAddress
getNameInfo
getNamespaceInfo
getZonefile
getAccountStatus
getAccountHistoryPage
getAccountAt
getAccountTokens
getAccountBalance
``` 
not `estimateTokenTransfer`

**Comment by friedger** [comment]
@larrysalibra @shreyasthiagaraj Are these the right methods, I don't see a use case for the ordinary app developer. I understand that it is interesting for a wallet/browser developer. Shall we provide it in a separate library?

**Comment by shreyasthiagaraj** [comment]
@friedger Wallets are apps too. I think we should support all methods and let developers figure out the use cases.

**Comment by friedger** [comment]
Fixed by #115
